modules/codegen/src/main/scala/com/twilio/swagger/codegen/generators/AkkaHttpServerGenerator.scala

```diff
@@ -112,7 +113,7 @@ object AkkaHttpServerGenerator {
                 ServerRoute(pathStr, httpMethod, operation)
               }
             }
-          }).sequenceU.map(_.flatten)
+          }).sequence.map(_.flatten)
```

Comment:
I believe this can be `.flatSequence`.

Reply:
I'm sure this is a resolvable error, but I'm getting:

```
Cannot prove that cats.data.EitherT[com.twilio.swagger.codegen.Logger,String,scala.collection.mutable.Iterable[com.twilio.swagger.codegen.ServerRoute]] <:< G[List[A]].
```
it looks like the `mutable.Iterable` is the problem and if I stick a `.toList` on line 115, then `flatSequence` works. Any thoughts on a better approach?

Reply:
Yeah I didn't look at the types carefully. I was assuming that was a monadic join. If it's the weird stdlib `.flatten` then what you have now is the best you can do.

Reply:
I'm thinking
```
diff --git a/modules/codegen/src/main/scala/com/twilio/swagger/codegen/generators/AkkaHttpServerGenerator.scala b/modules/codegen/src/main/scala/com/twilio/swagger/codegen/generators/AkkaHttpServerGenerator.scala
index e053afd..a2c7e72 100644
--- a/modules/codegen/src/main/scala/com/twilio/swagger/codegen/generators/AkkaHttpServerGenerator.scala
+++ b/modules/codegen/src/main/scala/com/twilio/swagger/codegen/generators/AkkaHttpServerGenerator.scala
@@ -104,17 +104,17 @@ object AkkaHttpServerGenerator {
       case ExtractOperations(paths) =>
         for {
           _ <- Target.log.debug("AkkaHttpServerGenerator", "server")(s"extractOperations(${paths})")
-          routes <- paths.map({ case (pathStr, path) =>
+          routes <- paths.traverse({ case (pathStr, path) =>
             for {
               _ <- Target.log.info("AkkaHttpServerGenerator", "server", "extractOperations")(s"(${pathStr}, ${path})")
               operationMap <- Target.fromOption(Option(path.getOperationMap), "No operations defined")
             } yield {
-              operationMap.asScala.map { case (httpMethod, operation) =>
+              operationMap.asScala.toList.map { case (httpMethod, operation) =>
                 ServerRoute(pathStr, httpMethod, operation)
               }
             }
-          }).sequence.map(_.flatten)
-        } yield routes
+          })
+        } yield routes.flatten
 
       case GetClassName(operation) =>
         for {
```
might be clearer, as it surfaces where we actually flatten the paths we get (from `paths`) and the operations (`operationMap.toScala.toList`) into a single `List[ServerRoute]`, should we ever discover we need to do more work before this information is lost.